AsaApi/Core/Public/API/ARK/PrimalStructure.h

```diff
@@ -2073,6 +2073,7 @@ struct APrimalStructureTurret : APrimalStructureItemContainer
     float& AimSpreadField() { return *GetNativePointerField<float*>(this, "APrimalStructureTurret.AimSpread"); }
     unsigned __int8& RangeSettingField() { return *GetNativePointerField<unsigned __int8*>(this, "APrimalStructureTurret.RangeSetting"); }
     unsigned __int8& WarningSettingField() { return *GetNativePointerField<unsigned __int8*>(this, "APrimalStructureTurret.WarningSetting"); }
+    unsigned __int8& AISettingField() { return *GetNativePointerField<unsigned __int8*>(this, "APrimalStructureTurret.AISetting"); }
```

Comment:
Please use `int8_t` instead of `__int8`
https://en.cppreference.com/w/cpp/types/integer